benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_vep.prm

```diff
@@ -0,0 +1,75 @@
+# This parameter file modifies the benchmark gerya_2019_vp.prm 
+# to include viscoelasticity.
+
+include $ASPECT_SOURCE_DIR/benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_vp.prm
+
+#  Global parameters
+set Dimension                              = 2
```

Comment:
You don't need this, correct? (it's already in the other file you include)

Reply:
yep, removed.